README.md

```diff
@@ -307,6 +307,40 @@ can then source these virtual environments using the instructions outlined
 a line such as this to each job destination:
 
     <env file="/path/to/shared/galaxy/venv" />
+    
+Using an external Grid Engine cluster
+-------------------------------------
+
+Almost things is as same as Slurm cluster.
+
+To use Grid Engine ( Sun Grid Engine, Open Grid Scheduler )
+ , one file and setup un environment variable are required:
+
+ 1. set environment SGE_ROOT
+ 2. create /var/lib/gridengine/default/common/act_qmaster file
+
+By default
+
+```
+-e SGE_ROOT=/var/lib/gridengine|
+-v $PWD/act_qmaster:/var/lib/gridengine/default/common/act_qmaster
+```
+
+In ***act_qmaster*** is something like this.
+
+```
+YOURGRIDENGINEMASTERHOST
+```
+
+Your Grid Engine need to accept job submit from container.
+
+If Grid Engine accept job submit from docker host,
+an easiest way is to use following docker option
+
+```
+--net=host
+```
+
```

Comment:
```--hostname SUBMIT_HOSTNAME``` works one environment.
But one environment can't work this even if SUBMIT_HOSTNAME is added 
we don't confirm all of requirements.

One complicated way but I don't know it works well because I don't test yet is using bridge interface container and docker host.